modules/gdnative/godot/string.cpp

```diff
@@ -538,6 +538,38 @@ godot_string GDAPI godot_string_get_slicec(const godot_string *p_self, wchar_t p
 	return result;
 };
 
+godot_array GDAPI godot_string_split(const godot_string *p_self, const godot_string *p_splitter) {
```

Comment:
Hey @karroffel  !

Are you sure that I don't need a conversion go through `Variant *` ?
The compiler warn me of the following error:
```cpp
modules/gdnative/godot/string.cpp:553:12: error: no viable overloaded '='
                proxy[i] = return_value[i];
                ~~~~~~~~ ^ ~~~~~~~~~~~~~~~
core/array.h:59:7: note: candidate function not viable: no known conversion from 'String' to 'const Array' for 1st argument
        void operator=(const Array &p_array);

```